web/index.php

```diff
@@ -63,9 +63,12 @@ function generate_epics_based_on_starting_label($starting_label, $number_of_iter
     $epics = generate_epics_based_on_starting_label($starting_label, $number_of_iterations);
```

Comment:
More a suggestion, but we should probably rename this variable something like `iterations` since we're not using labels anymore.